cunit/aaa-db.testc

```diff
@@ -19,7 +19,14 @@ struct binary_result
     size_t datalen;
 };
 
-static const char *backend = CUNIT_PARAM("skiplist,flat,twom,twoskip");
+#ifdef USE_CYRUSDB_SQL
+// #define BACKENDS "skiplist,flat,twom,twoskip,sql"
```

Comment:
does this work?  cunit.pl says:

```
   # Also scan for variable declarations of the form
   #
   # [static] [const] char * foo = CUNIT_PARAM("stringliteral");
   #
   # and make them a parameter for the containing test suite.
```

Or does cunit expect to parse those values after the preprocessor has run?